recipe/meta.yaml

```
@@ -37,17 +39,58 @@ requirements:
     - fsspec >=2021.05.0
     - huggingface_hub >=0.7.0
     - packaging
-    - responses <0.19
-    - cookiecutter # to populate metric template. Required by CLI: https://github.com/huggingface/evaluate/blob/v0.3.0/src/evaluate/commands/evaluate_cli.py#L6
+  run_constrained:
+    # Extra: evaluator
+    - scipy >=1.7.1
+    # Extra: template
+    - gradio >=3.0.0
+    # Extra: quality
+    - black >=22.0,<23
+    - flake8 >=3.8.3
+    - isort >=5.0.0
+    - pyyaml >=5.3.1
+    # Extra: tensorflow
+    - tensorflow >=2.2.0,!=2.6.0,!=2.6.1
+    # Extra: tensorflow_gpu
+    - tensorflow-gpu >=2.2.0,!=2.6.0,!=2.6.1
```

```suggestion
    # Extra: tensorflow
    - tensorflow-base >=2.2.0,!=2.6.0,!=2.6.1
```
This covers both gpu and cpu